File: common/src/test/java/dev/number6/SlackAssertions.java
package dev.number6;

import dev.number6.slack.OkHttpRequestAdaptor;
import org.assertj.core.api.Assertions;

public class SlackAssertions extends Assertions {

    public static RequestPortAssert assertThat(OkHttpRequestAdaptor callRequest) {
        return new RequestPortAssert(callRequest);
    }
}

File: common/src/test/java/dev/number6/SlackAssertions.kt
package dev.number6

import dev.number6.slack.OkHttpRequestAdaptor
import org.assertj.core.api.Assertions

object SlackAssertions : Assertions() {
    fun assertThat(callRequest: OkHttpRequestAdaptor): RequestPortAssert {
        return RequestPortAssert(callRequest)
    }
}